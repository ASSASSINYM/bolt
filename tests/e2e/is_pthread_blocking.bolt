void main(){
  let x = 5;
  finish {
    async{
      x := 6;
     printf("Value of x: %d\n", x)
     (* Should print if blocking *)
    }
    x
  };
x
}