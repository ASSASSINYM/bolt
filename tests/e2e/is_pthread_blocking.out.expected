Value of x: 6\n